salvadorTests/FetchImage_Tests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FetchImage_Tests.swift
+//  salvadorTests
+//
+//  Created by Geury Roustand on 22.02.23.
+//
+
+import XCTest
+
+@testable import salvador
+
+final class FetchImage_Tests: XCTestCase {
+    
+    var createImage: CreateImage!
+    var viewModel : GenerateImageViewModel!
+    
+    @MainActor override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        createImage = CreateImage()
+        viewModel = GenerateImageViewModel()
+    }
+    
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        createImage = nil
+        viewModel = nil
+    }
+    
+    func testGenerateImage_SuccessFulResponsw() async throws {
+        let prompt = "Create a mango eating an apple with sunglases on the beach"
+        
+        let apiKey = await viewModel.loadJson(fileName: "APISecret")
```

Comment:
You could also put the `loadJson` to a `fileLoaderService` and create as well a mock for it so you are not relying on loading an actual real file.